tests/test_mesh.py

```diff
@@ -343,23 +330,51 @@ def arithmetic_statmesh_setup(self):
 
     def test_mutliply_vector_by_scalar(self):
         self.arithmetic_mesh_vector_setup()
-        scalar = 12.0
-        test_vector_tag = self.mesh_1_vector.mesh.tag_get_handle(self.vector_tag_name)
-        test_data = self.mesh_1_vector.mesh.tag_get_data(test_vector_tag, self.volumes1)
+        scalar = 12.0 
         exp_res = [[12.0, 24.0], [18.0, 30.0], [-12.0, -18.0], [-24.0, -30.0]]
-        obs_res = scalar*test_data
+        obs_res = scalar*self.mesh_1_vector.testing[:] 
         assert_array_almost_equal(exp_res, obs_res)
     
     def test_multiply_vector_tag_by_scalar_tag(self):
+        self.arithmetic_mesh_vector_setup()
+        exp_res = [[12.0, 24.0], [9.0, 15.0], [-3.0, -4.5], [-3.0, -3.75]] 
+        scalar_data = self.mesh_1_vector.test_scalar_tag[:].reshape((4,1))
+        vector_data = self.mesh_1_vector.testing[:]
+        obs_res = scalar_data * vector_data
+        assert_array_almost_equal(exp_res, obs_res)
+
+    """
+    def test_multiply_vector_tag_by_scalar_tag_element(self):
         self.arithmetic_mesh_vector_setup()
         test_scalar_tag = self.mesh_1_vector.mesh.tag_get_handle(self.scalar_tag_name)
         test_vector_tag = self.mesh_1_vector.mesh.tag_get_handle(self.vector_tag_name)
         scalar_data = self.mesh_1_vector.mesh.tag_get_data(test_scalar_tag, self.volumes1)
         vector_data = self.mesh_1_vector.mesh.tag_get_data(test_vector_tag, self.volumes1)
-        exp_res = [[12.0, 24.0], [9.0, 15.0], [-3.0, -4.5], [-3.0, -3.75]]
-        obs_res = scalar_data*vector_data
-        assert_array_almost_equal(exp_res, obs_res)
+        index = 0
+        exp_res = [[[12.0, 24.0], [18.0, 30.0], [-12.0, -18.0], [-24.0, -30.0]],
+                   [[6.0, 12.0], [9.0, 15.0], [-6.0, -9.0], [-12.0, -15.0]],
+                   [[3.0, 6.0], [4.5, 7.5], [-3.0, -4.5], [-6.0, -7.5]],
+                   [[1.5, 3.0], [2.25, 3.75], [-1.5, -2.25], [-3.0, -3.75]]]
+        for result in exp_res:
+            assert_array_almost_equal(result, scalar_data[index]*vector_data)
+            index += 1
+
 
+    def test_multiply_vector_tag_by_vector_tag_element(self):
```

Comment:
This test isn't really testing anything in PyNE

Reply:
So can we remove this test?

Reply:
Or should I change it to implement the `mesh.tagname[:]` function

Reply:
I think the latter would be best - implement it using the slice operators, although that's possibly somewhat redundant to just testing the slice operators themselves (which may not be happening?)

Reply:
Yeah I see what you're saying. Maybe just drop all of the related tests and opt for one (or two) tests with a name like `test_slice_operator_vector()` and `test_slice_operator_scalar()`?